src/app/home/home.component.ts

```diff
@@ -172,6 +189,11 @@ export class HomeComponent implements AfterViewInit {
         }
     }
 
+    /**
+     * Apply the chosen suggestion in the editor.
+     * @param textMarkingIndex {number} the index of the chosen TextMarking
+     * @param suggestionIndex {number} the index of the chosen Suggestion of the above TextMarking
```

Comment:
```suggestion
     * @param {number} textMarkingIndex the index of the chosen TextMarking
     * @param {number} suggestionIndex the index of the chosen Suggestion of the above TextMarking
```
lets keep it consistent as the example above